actions/poweroff/v1/poweroff.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+    "fmt"
+    "log"
+    "os/exec"
+    "runtime"
+)
+
+// Dont have global variable !
+
+func main() {
+    // Checking the machine system
+    if (runtime.GOOS == "linux") {
+        LinuxShutdown()
+    } else if (runtime.GOOS == "windows") {
```

Comment:
We do not ship actions who can run on windows but only on Linux as specified in the README for this repository. Just a question, are you running tink-worker and netbooting a Windows OS instead of OSIE/Hook?